samples/tools/cli.py

```diff
@@ -57,6 +58,12 @@ def build_arg_parser():
                         required=False,
                         action='store',
                         help='Password to use when connecting to host')
+
+    parser.add_argument('--no-ssl',
```

Comment:
I would pass a boolean argument. Having a int argument is not very intuitive. How about make a boolean parameter and with default value to False, if user want to disable ssl verification, they can pass True?